src/chrome/content/rules/rsdn.org.xml

```diff
@@ -1,21 +1,27 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://blogs.rsdn.org/ => https://blogs.rsdn.org/: (35, 'Unknown SSL protocol error in connection to blogs.rsdn.org:443 ')
-Fetch error: http://track.rsdn.org/ => https://track.rsdn.org/: (35, 'Unknown SSL protocol error in connection to track.rsdn.org:443 ')
-Fetch error: http://tc.rsdn.org/ => https://tc.rsdn.org/: (35, 'Unknown SSL protocol error in connection to tc.rsdn.org:443 ')
-
-rsdn.org mixed content
-www.rsdn.org mixed content
-Example URLs that are broken from mixed content blocking:
-https://rsdn.org/?http://confluence.jetbrains.com/display/Nitra
-https://www.rsdn.org/?http://confluence.jetbrains.com/display/Nitra
+rsdn.org
 -->
-<ruleset name="rsdn.org (partial)" default_off="failed ruleset test">
-	<target host="files.rsdn.org" />
+<ruleset name="rsdn.org (partial)">
+	<target host="rsdn.org" />
+	<target host="www.rsdn.org" />
 	<target host="blogs.rsdn.org" />
-	<target host="track.rsdn.org" />
-	<target host="tc.rsdn.org" />
+	<target host="files.rsdn.org" />
+
+	<test url="http://rsdn.org/?" />
+	<test url="http://rsdn.org/#" />
+	<test url="http://rsdn.org/?q" />
+
+	<test url="http://www.rsdn.org/?" />
+	<test url="http://www.rsdn.org/#" />
+	<test url="http://www.rsdn.org/?q" />
+
+	<test url="http://blogs.rsdn.org/?" />
+	<test url="http://blogs.rsdn.org/#" />
+	<test url="http://blogs.rsdn.org/?q" />
+
+	<test url="http://files.rsdn.org/?" />
+	<test url="http://files.rsdn.org/#" />
+	<test url="http://files.rsdn.org/?q" />
```

Comment:
In this case, all test urls are redundant. You just drop them.

Reply:
Or you can add real test urls that point to some content.

Reply:
Ok.